src/pudl/constants.py

```diff
@@ -1145,6 +1147,7 @@
 contract_type_eia923 = {
     'C': 'Contract - Fuel received under a purchase order or contract with a term of one year or longer.  Contracts with a shorter term are considered spot purchases ',
     'NC': 'New Contract - Fuel received under a purchase order or contract with duration of one year or longer, under which deliveries were first made during the reporting month',
+    'N': 'New Contract - see NC code. This abbreviation existed only in 2008 before being replaced by NC.',
```

Comment:
I agree that this case is completely unambiguous and the outputs should be standardized. I'll remove it and leave a comment about the value for the future. 
I added to this dict because I wasn't sure how it was used (I found zero references to it). In the datapackage validation, there are foreign key constraints on some nearly-raw (unmapped) data columns, so I guessed the purpose was to warn when raw values differed from EIA expectations.